src/cms/templates/dashboard/admin_dashboard.html

```diff
@@ -1,5 +1,134 @@
 {% extends "_base.html" %}
 {% load i18n %}
+{% block javascript_head %}
+<script src="{% static 'js/DeleteRowTable.js' %}"></script>
+{% endblock %}
 {% block content %}
     Admin Dashboard
-{% endblock %}
\ No newline at end of file
+{% load static %}
```

Comment:
```suggestion
```
This is too late, because you use the `{% static %}`-tag earlier.
Please place all `{% load %}`-tags at the top of the page.